distage/distage-framework-docker/src/main/scala/izumi/distage/docker/Docker.scala

```diff
@@ -25,7 +30,7 @@ object Docker {
     def address: InetAddress
     def host: String
 
-    override final def toString: String = address.getHostAddress
+    override final def toString: String = host
```

Comment:
@Caparow I think it makes sense to print IPV6 addresses with the surrounding `[]` brackets, however I'm not sure if there was a reason not to do this in the first place ☝️

Reply:
@neko-kai I've thought of printing V6 addresses without any additional changes, just if it would be returned by InetAddress. But it's safe to print it with `[]`